src/core/plugins/json-schema-5-samples/fn/index.js

```diff
@@ -56,10 +56,12 @@ const numberContracts = [
 ]
 const stringContracts = ["minLength", "maxLength"]
 
-const liftSampleHelper = (oldSchema, target, config = {}) => {
+export const mergeJsonSchema = (target, oldSchema, config = {}) => {
```

Comment:
Renamed the function and changed the order of arguments to match the `merge` function in `JSON Schema 2020-12 samples` plugin

Reply:
Looks good, I'd maybe rename `oldSchema` to `source`. `oldSchema` is  confusing and doesn't mean nothing.